### PR TITLE
Cyclegan tweaks

### PR DESCRIPTION
**Pull request by brettkoonce**

### CODE REVIEW

**Comment by saeta** [comment]
Thank you @brettkoonce !

File: CycleGAN/main.swift [hunk 1]
```diff
@@ -15,14 +15,45 @@
...
+	        remoteRoot: location.deletingLastPathComponent(), localStorageDirectory: directory)
+	}
+
+	datasetFolder = DatasetUtilities.defaultDirectory.appendingPathComponent("CycleGAN", isDirectory: true)
+	downloadZebraDataSetIfNotPresent(to: datasetFolder)
+	trainFolderA = datasetFolder.appendingPathComponent("horse2zebra/trainA")
+	trainFolderB = datasetFolder.appendingPathComponent("horse2zebra/trainB")
+	testFolderA = datasetFolder.appendingPathComponent("horse2zebra/testA")
+	testFolderB = datasetFolder.appendingPathComponent("horse2zebra/testB")
+}
```
Comment (BradLarson) [inline CycleGAN/main.swift:56]: Minor formatting issue: it looks like the indentation here is using tabs, and not four spaces, so the code block is getting indented in a way that is different from the remainder of the code.

File: CycleGAN/main.swift [hunk 2]
```diff
@@ -15,14 +15,45 @@
...
-let datasetFolder = URL(fileURLWithPath: options.datasetPath, isDirectory: true)
-let trainFolderA = datasetFolder.appendingPathComponent("trainA")
-let trainFolderB = datasetFolder.appendingPathComponent("trainB")
-let testFolderA = datasetFolder.appendingPathComponent("testA")
-let testFolderB = datasetFolder.appendingPathComponent("testB")
+var datasetFolder: URL
+var trainFolderA: URL
+var trainFolderB: URL
+var testFolderA: URL
+var testFolderB: URL
```
Comment (BradLarson) [inline CycleGAN/main.swift:26]: Can these still be `let`s, because they are set in both paths of the conditional statement and not reassigned?

Comment (brettkoonce) [reply CycleGAN/main.swift]: updated!

File: CycleGAN/main.swift (hunk 1 above)
Comment (brettkoonce) [reply CycleGAN/main.swift]: tweaked!

**Pull request merged**